pkg/controller/kogitoinfra/kafka/resource.go

```diff
@@ -76,7 +76,7 @@ func CreateRequiredResources(kogitoInfra *v1alpha1.KogitoInfra) (resources map[r
 						"offsets.topic.replication.factor":         defaultReplicas,
 						"transaction.state.log.min.isr":            2,
 						"transaction.state.log.replication.factor": defaultReplicas,
-						"auto.create.topics.enable":                false,
+						"auto.create.topics.enable":                true,
```

Comment:
If this is set to `true`, kafka topics could be created automatically, but not owned by the kogito custom resources.

Reply:
Strimzi is not creating the Kafka Topics CR. This is just a flag to auto-create them in the Kafka cluster since Kogito Runtimes and Jobs Service don't create the Kafka Topics CR like Data Index does. Also, I guess we could drop this feature of creating the Topics CR by ourselves and just let Kafka to create them. WDYT?

Reply:
It makes sense if the kafka topics are not supposed to be deleted when data index is uninstalled.